src/App/Fossa/FossaAPIV1.hs

```diff
@@ -377,6 +383,28 @@ archiveBuildUpload apiOpts archiveProjects = runEmpty $
         req POST (archiveBuildURL baseUrl) (ReqBodyJson archiveProjects) bsResponse (baseOpts <> opts)
     pure (responseBody resp)
 
+---------- license-scan build queueing. This Endpoint ensures that after a license-scan is uploaded, it is scanned.
+
+licenseScanFinalizeUrl :: Url 'Https -> Url 'Https
+licenseScanFinalizeUrl baseUrl = baseUrl /: "api" /: "license_scan" /: "finalize"
+
+-- TODO: /license_scan/finalize just returns a 201 if there's a success. No need to parse the body
+licenseScanFinalize ::
```

Comment:
[Optional] I can't pin down why I'm not a fan of `licenseScanFinalize`, but maybe `queueBuildForLicenceScanUploads` is more apt?